lib/repo/git_repository.rb

```diff
@@ -391,47 +472,59 @@ def get_users(permissions)
 
     def get_permissions(user_id)
 
-      #if @repos_admin # Are we admin?
-      # Adds a user with given permissions to the repository
+      if @repos_admin # Are we admin?
+        # Adds a user with given permissions to the repository
+        ga_repo = Gitolite::GitoliteAdmin.new(Repository.conf[:REPOSITORY_PERMISSION_FILE])
 
-      ga_repo =
-          Gitolite::GitoliteAdmin.new(Repository.conf[:REPOSITORY_STORAGE])
-      repo = ga_repo.config.get_repo(self.get_repos.workdir.split('/').last)
+        # Sync the admin repo
+        ga_repo.update
+        repo = ga_repo.config.get_repo(self.get_repos.workdir.split('/').last)
 
-      # Gets permissions of a particular user
-      repo.permissions[0].each do |perm|
-        if(repo.permissions[0][perm[0]][""].include? user_id)
-          return self.class.__translate_perms_from_file(perm[0])
+        # Gets permissions of a particular user
+        repo.permissions[0].each do |perm|
+          if(repo.permissions[0][perm[0]][""].include? user_id)
+            return self.class.__translate_perms_from_file(perm[0])
+          end
         end
-      end
-      raise UserNotFound.new(user_id + " not found")
 
-      #else
-      #  raise NotAuthorityError.new("Unable to modify permissions: Not in authoritative mode!")
-      #end
+        raise UserNotFound.new(user_id + " not found")
+
+      else
+        raise NotAuthorityError.new("Unable to modify permissions: Not in authoritative mode!")
```

Comment:
Line is too long. [95/80]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.